cypress/integration/crud.spec.ts

```diff
@@ -0,0 +1,50 @@
+const book_title = 'The Little Schemer';
+
+describe('CRUD Actions', () => {
+	context('Valid Data', function(){
+		beforeEach(() => {
+			cy.visit('/#/examples/crud');
+			cy.get('.add').click();
```

Comment:
I can add the title attribute. I was thinking of adding "data-action=add" instead of title. What do you think?

Reply:
Instead of `data-action` I would suggest using `data-testid="add-crud"`, this is also what cypress encourages - https://docs.cypress.io/guides/references/best-practices.html.